yt_dlp/extractor/scrippsnetworks.py

```diff
@@ -113,6 +114,9 @@ class ScrippsNetworksIE(InfoExtractor):
             'timestamp': 1475678834,
             'upload_date': '20161005',
             'uploader': 'SCNI-SCND',
+            'duration': 29.995,
+            'chapters': [{'start_time': 0.0, 'end_time': 29.995, 'title': '<Untitled Chapter 1>'}],
```

Comment:
Is this expected behaviour?

Reply:
yes. downloads tests used to ignore the `chapters` field back when this test was originally added